source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -20,6 +20,10 @@ Along with the deprecation announcement, MinIO also announced that the feature w
 As of :minio-release:`RELEASE.2022-10-29T06-21-33Z`, the MinIO Gateway and the related filesystem mode code have been removed.
 Deployments still using the `standalone` or `filesystem` MinIO modes that upgrade to :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later receive an error when attempting to start MinIO.
 
+.. cond:: linux
+
+   For deployments running in a container, see the `Container - Migrate from Gateway or Filesystem Mode <https://min.io/docs/minio/container/operations/install-deploy-manage/migrate-fs-gateway.html>`__ tutorial instead.
```

Comment:
I was hasty with the review button, should the `__` be there?

(But also yes this is very needed.)

Reply:
Need a trailing underscore or two after links that don't have a role. So, yes, they need to be there.

Reply:
When it's one or two underscores I have not yet figured out. 😄

Reply:
The two underscores makes it an "anonymous" link. Without that it becomes a target link, and if any other link on the page matches that URL it breaks.

I find this annoying to keep track of so I just make all full URLs to be `__` by default.